src/test_mailroom.py

```diff
@@ -0,0 +1,83 @@
+"""Test mailroom module for managing donation history list, emails."""
+
+import pytest
+
+
+def test_create_sorted_list():
+    """Test the create_sorted_list function."""
+    from mailroom import create_sorted_list
+    donors = {'jordan schatzman': [100, 200, 400, 50.6],
+              'rick valenzuela': [500, 300, 100, 50, 1.24],
+              'sally johnson': [50, 100, 1, 2, 99],
+              'jane': [39, 23, 532, 2432],
+              'bob': [89, 23, 12, 3.45643],
+              'XXXX TEXT': [0]}
+
+    test_list = create_sorted_list(donors)
```

Comment:
You should generally limit yourself to only one assert statement per test function.

Consider how you could do this kind of testing with parametrization and/or a fixture.